src/ROOTReader.cc

```diff
@@ -15,6 +15,9 @@
 #include <memory>
 
 namespace podio {
+  
+// todo: see https://github.com/AIDASoft/podio/issues/290
+ROOTReader::~ROOTReader() {} // NOLINT
```

Comment:
```suggestion
ROOTReader::~ROOTReader() { // NOLINT(modernize-use-equals-default)
}
```
Not sure where the `NOLINT` has to go in this case, this is really just fixing the `clang-format` issues (hopefully)